docs/src/pages/playground.tsx

```diff
@@ -0,0 +1,221 @@
+import React, { useEffect, useRef, useState } from 'react'
+import Layout from '@theme/Layout'
+import sdk from '@stackblitz/sdk'
+import styles from './playground.module.css'
+import { useColorMode } from '@docusaurus/theme-common'
+import { usePluginData } from '@docusaurus/useGlobalData'
+
+interface Project {
+	categoryTitle: string
+	subCategory: Array<CategoryItems>
+}
+
+interface CategoryItems {
+	name: string
+	templates: Array<Templates>
+}
+
+interface Templates {
```

Comment:
Singular name would be more appropriate here, otherwise it implies an array. `Templates` -> `Template`